### ISSUE TITLE
Mouse unavailable - Retroarch 1.8.4 Mac

### ISSUE THREAD

**Issue by groelk** [libretro/libretro-uae#250]
Hi,
PUAE is an amazing core. It's great to be able to make Amiga's best games alive again.
No issues at all when using it on Android. But I can't get it to work with any type of mouse on Retroarch's current Mac version. Games open up well, yet there's no way the pointer will move with a mouse or trackpad, only with the keyboard. I think I tried all available settings...
Is this a known issue, and are there any ways to fix it?
Many thanks!

**Comment by rsn8887** [comment]
I don’t think this is a UAE core issue. I think this is a frontend issue and should be raised in the retroarch repository. Unless you can get the mouse to work in any other core on MacOS?!?

**Comment by groelk** [comment]
Thanks! Just posted about this issue on the RA repo as you recommended.

**Comment by gouchi** [comment]
- [RA issue](https://github.com/libretro/RetroArch/issues/10168)

**Comment by OctopusButtons** [comment]
I have the same issue.  There is no way to get any mouse device working in-game, I tried both PSX Beetle, Nestopia, etc.  I tried 4 different devices on Mac (MacBook Pro trackpad, wireless Bluetooth trackpad, Bluetooth mouse, standard generic USB mouse), I tried every single Mouse Index setting both while repeatedly toggling “Grab Mouse Input” (just in case) and while also toggling Game Focus on and off (just in case), I tried every virtual  control setting (e.g. in Time Crisis for PlayStation I tried “Guncon” and other options).  I deleted configs.  I redid the binds.  I unbinded and rebinded.  I tried multiple Users, 1 and 2, just in case.

There is no way to make the crosshair move at all, even a single pixel, in Time Crisis on PlayStation (Beetle core, also the PCArmed core).

All these mouses work fine in the OS and in the Retroarch front end menu.

As soon as I change to a controller input, I can move the controller cursor with Dpad or keyboard arrows.  But absolute no mouse movement in-game.  But my mouse works OK as a lightgun cursor mover in MAME (outside of Retroarch).
